ui/uihandler.go

```diff
@@ -472,13 +453,17 @@ func Serve(port int, debugpath string, uiPath string) {
 		}
 		http.HandleFunc(uiPath, Handler)
 		fmt.Printf("UI starting - visit:\nhttp://localhost:%d%s\n", port, uiPath)
-		logoPath = uiPath + logoURI
-		http.HandleFunc(logoPath, LogoHandler)
-		chartJSPath = uiPath + chartjsURI
-		http.HandleFunc(chartJSPath, ChartJSHandler)
-		// Use relative paths after that (in the html template):
-		logoPath = "./" + logoURI
-		chartJSPath = "./" + chartjsURI
 	}
+
+	// Serve static contents in the ui/static dir.
+	// We use directory relative to this file to find the static contents, so no matter where is the working dir, static dir could be found.
+	_, filename, _, ok := runtime.Caller(0)
```

Comment:
so file here is fully qualified source file at build time?
where would we place the files in docker ?
how is the content type set?

Reply:
Yes, and modified Dockerfile too. Content type is automatically detected by file extension.

Reply:
sorry I had missed the docker change

in which case runtime.Caller() would be not available ? is there a chance stripped binaries don't have the source code information for instance?

edit: I checked that strip for instance makes it fail...